tools/openssl_install_script/install-openssl.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
```

Comment:
Technically this is a helper script. Not sure if you want it in Gen2 when we can prescribe versioning (basically require OpenSSL 3.0 to run).

Reply:
In later PR, will remove